src/App.jsx

```diff
@@ -1,42 +1,55 @@
 import React, { useState } from "react";
-import "aframe";
-import "aframe-event-set-component";
-import "aframe-orbit-controls";
 
-import "./Aframe/loader.js";
-import "./Aframe/buttons-check.js";
-import "./Aframe/render-2d-clipplane";
-import "./Aframe/cursor-listener";
-
-import "./styles/main.scss";
-
-import { SelectorProvider } from "./context/selector-context.js";
+import config from "./assets/config.json";
+import haline from "./assets/colormaps/haline.png";
+import thermal from "./assets/colormaps/thermal.png";
 
 import Header from "./components/Header";
 import InfoText from "./components/InfoText";
 import ModelSelector from "./components/ModelSelector";
+import VolumeViewerWrapper from "./components/VolumeViewerWrapper";
 import Instructions from "./components/instructions/Instructions";
 import Footer from "./components/Footer";
-import VolumeViewerWrapper from "./components/volume-viewer/VolumeViewerWrapper.jsx";
-import { ControlsProvider } from "./context/controls-context.js";
+import "./styles/main.scss";
+
+const ALL_COLOR_MAPS = {
+  Haline: haline,
+  Thermal: thermal,
+};
+const BUTTONS = {
+  season: config.season,
+  tide: config.tide,
+  measurement: config.measurement,
+};
 
 export default function App() {
   const [controlsVisible, setControlsVisible] = useState(false);
 
+  const [selection, setSelection] = useState({
+    season: BUTTONS.season[0],
+    tide: BUTTONS.tide[0],
+    measurement: BUTTONS.measurement[0],
+  });
+  const [colorMap, setColorMap] = useState(ALL_COLOR_MAPS.Haline);
+
   return (
     <div id="visualizer">
       <Header />
-
-      <SelectorProvider>
-        <ControlsProvider>
-          <InfoText />
-          <ModelSelector
-            toggleControls={() => setControlsVisible(!controlsVisible)}
-          />
-          <VolumeViewerWrapper controlsVisible={controlsVisible} />
-        </ControlsProvider>
-      </SelectorProvider>
-
+      <InfoText selection={selection} />
+      <ModelSelector
+        BUTTONS={BUTTONS}
+        ALL_COLOR_MAPS={ALL_COLOR_MAPS}
```

Comment:
Same question as on the volume viewer of props vs consts for these items.

Reply:
Actually I think I'll make a `constants.js` file like with volume viewer. I should still use the config.json file though, right? That's potentially something that could be automated down the line?

Reply:
I think the difference in this case is largely perception.  When I see a constants file, I think "I shouldn't touch that", when I see a config file, I think "ooooh - let's go fiddle with that".

Reply:
Yeah I'm inclined to agree. Right now I'm using constants to parse everything in the config files for the app. Constants can pretty much be left untouched as the app changes